src/Settings_Helper.php

```diff
@@ -101,12 +96,11 @@ public function add_fields( $fields, $setting_tab, $neighboring_field = null, $a
 
 	}
 
-
 	/**
 	 * Remove a field from one of the tabs in WP Admin > Events > Settings
 	 *
 	 * @param string $field_key   Option key for your setting. Example: 'fancyOptionName'.
-	 * @param string $setting_tab Settings tab where this will be added. Example: 'display'.
+	 * @param string $setting_tab Settings tab from where this will be removed. Example: 'display'.
 	 */
 	public function remove_field( $field_key, $setting_tab ) {
 		$this->remove_fields[$setting_tab][] = $field_key;
```

Comment:
```suggestion
		$this->remove_fields[ $setting_tab ][] = $field_key;
```
We normally add spaces around array keys when we use vars as keys. If we use the key directly (Like `$array['key']`) then it's not required to add spaces.

Reply:
Oh cool, I didn't know. Thanks for teaching me something new. :)